libs/encryption-sdk/src/main/java/org/opensearch/encryption/frame/core/EncryptionMetadata.java

```diff
@@ -45,6 +45,7 @@
 import java.util.Map;
 
 @SuppressWarnings({ "rawtypes" })
+// Move this to EncryptionHandler
```

Comment:
I don't think we should. If we move it there then we won't have a single encapsulation for encryption metadata. We will have to define new metadata schema and creation of this metadata will add redundant code to create metadata.